NoSlip process fix

rubenzorrilla opened the pull request:
Now the NoSlip BC imposition is done during the ExecuteInitializeSolutionStep() (this avoids the "corner" nodes velocity overwrite). Besides, the new "experimental" processes have been included.

rubenzorrilla committed:
NoSlip process fix

Now the NoSlip BC imposition is done during the ExecuteInitializeSolutionStep() (this avoids the "corner" nodes velocity overwrite). Besides, the new "experimental" processes have been included.

RiccardoRossi commented on applications/FluidDynamicsApplication/python_scripts/apply_noslip_process.py:
```diff
@@ -1,34 +1,75 @@
+
+        self.aux_processes = []
+
+        import experimental_assign_value_process
```
just to say that maybe the vector version of the "experimental" processes would save you some writing

RiccardoRossi commented on applications/FluidDynamicsApplication/python_scripts/apply_noslip_process.py:
```diff
@@ -1,34 +1,75 @@
-##all the processes python processes should be derived from "python_process"
-class ApplyNoSlipProcess(Process):
+
+class ApplyNoSlipProcess(KratosMultiphysics.Process):
```
i think that if you derive this from the vector version of the experimental process this will be almost empty

rubenzorrilla committed:
Usign experimental_assign_vector_process

Instead of using three independent experimental_assign_value_process for each component a unique experimental_assign_vector_process is used.

rubenzorrilla commented:
@RiccardoRossi  now is much more clean.

RiccardoRossi reviewed (approved):
+1
